tests/integration/golang/mlflow/run/delete_test.go

```diff
@@ -39,33 +39,39 @@ func (s *DeleteRunTestSuite) SetupTest() {
 	expFixtures, err := fixtures.NewExperimentFixtures(helpers.GetDatabaseUri())
 	assert.Nil(s.T(), err)
 	s.experimentFixtures = expFixtures
+}
+
+func (s *DeleteRunTestSuite) Test_Ok() {
+	defer func() {
+		assert.Nil(s.T(), s.runFixtures.UnloadFixtures())
+		assert.Nil(s.T(), s.experimentFixtures.UnloadFixtures())
```

Comment:
i think all the fixtures inherit Base/UnloadFixtures() -- so only need one call

Reply:
yep yep, you are totally right. it seems I just missed this place.